recipes/tests.rb

```diff
@@ -136,11 +136,27 @@
       command "rpm -qa | grep libfabric && rpm -qa | grep efa-"
       user node['cfncluster']['cfn_cluster_user']
     end
+    execute 'check intel mpi installed' do
```

Comment:
Leaving the comment here but it refers to L132: Aren't we installing EFA and intel mpi in cn regions too?

Reply:
yes, I've removed not `cn` check